client/js/_author/components/assessments/navigation_bar_content.jsx

```diff
@@ -43,14 +46,13 @@ export default class NavigationBarContent extends React.Component {
           { this.publishButton() }
           {
             this.props.isPublished ?
-              <Link
+              <button
```

Comment:
No, it still has all of the same classes applied. I think Link just renders a button anyway. Good thought though.